core/src/main/resources/config.yml

```
@@ -120,3 +120,16 @@ moderation:
     
   # Last message line shown on ban/temp-ban kick screen
   appeal-msg: ''
+  
+fireworks:
+  # Post-match fireworks fire above the winners of the current match
+  post-match:
+    enabled: true
+    number: 5
+    delay: 40 # ticks
+    frequency: 40 # ticks
+    iterations: 15
+    power: 2
+  # Goals spawn fireworks when objectives are completed
+  goals:
+    enabled: true
```

```yml
fireworks:
     enabled: true
```

---

I would keep it as:
```suggestion
fireworks:
  post-match: false
  goals: true
```
(post-match default being false)